### PR TITLE
Code Review

### PR DESCRIPTION
**Pull request by 28919926**
Add changes on the index.html and css files.
Fix linter errors.
Add changes.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -185,3 +185,16 @@ button:active {
...
+.form {
+  display: flex;
+  width: 322px;
+height: 813px;
+left: -2px;
+  background-image: url(/images/imagenav.svg);
+  background-repeat: no-repeat;
+}
+
+#name-form {
```
Comment (28919926) [inline style.css:198]: Kindly don't use id attribute i.e #name-form if it's not necessary its a good practice to stick to classes unless required.

**Review by 28919926 — COMMENTED** [review]
**Hi Desmond[@28919926](https://github.com/28919926) ,**

Your project is almost there for completion!! 
You've done a commendable job👏👏👏  there are few changes that you need to work on.

Please check on the required changes.


Highlights
✅ Correct use of Github workflow
✅ No linter error
✅ README is okay
✅ Good PR Description👍

Optional suggestions
N/A

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

File: style.css (hunk 1 above)
Comment (christianbiring1) [reply style.css:198]: [OPTIONAL]
- [ ] Always make sure you explain `why` the changes are suggested. You can even provide the link to the resource that will explain better why the changes were suggested.

**Review by christianbiring1 — APPROVED** [review]
Hi @28919926 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 
nicee

## Highlights
- [ ] Use Github inline comments for the review :+1: 
- [ ] The comments are addressed in a friendly and polite way :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (28919926) [reply style.css:198]: okay i will do that and thanks

**Pull request merged**